.github/labeler.yml

```
@@ -2,19 +1,20 @@
+"Type: Docs":
 - changed-files:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)